### PR TITLE
make bootstrap script aware of vsc-install for offline installation

### PR DESCRIPTION
**Pull request by boegel**
inspired by the modifications made by the OpenHPC community to an old version of the bootstrap script, cfr.:
- https://github.com/openhpc/ohpc/commit/aac22f193c64f42acbc82ff9bd70f0f5c9bc74b8
- https://github.com/openhpc/ohpc/commit/3f9442a6b53938e1d31cda70dae7d7a66e9c2827
- https://github.com/openhpc/ohpc/commit/4f14bcf95254569d7023e634743d3c4279e8bf12

edit: TODO:
- [ ] update docs (http://easybuild.readthedocs.io/en/latest/Installation.html#bootstrapping-using-supplied-source-tarballs)

### CODE REVIEW

**Comment by boegel** [comment]
this only impacts bootstrapping with provided sources (cfr. http://easybuild.readthedocs.io/en/latest/Installation.html#bootstrapping-using-supplied-source-tarballs)

@wpoely86 please review?

File: easybuild/scripts/bootstrap_eb.py [hunk 1]
```diff
@@ -66,7 +66,8 @@
 PYPI_SOURCE_URL = 'https://pypi.python.org/packages/source'
 
 VSC_BASE = 'vsc-base'
-EASYBUILD_PACKAGES = [VSC_BASE, 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']
+VSC_INSTALL = 'vsc-install'
+EASYBUILD_PACKAGES = [VSC_BASE, VSC_INSTALL, 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']
```
Comment (wpoely86) [inline easybuild/scripts/bootstrap_eb.py]: don't you need first install then base?

Comment (boegel) [inline easybuild/scripts/bootstrap_eb.py]: ah, yes, crap, I'm making the same mistake OpenHPC did, cfr. https://github.com/openhpc/ohpc/commit/3f9442a6b53938e1d31cda70dae7d7a66e9c2827 ;)

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**